lib/Net/Async/Blockchain/ETH.pm

```diff
@@ -110,11 +112,31 @@ update the C<accounts> variable every 10 seconds
 async sub update_accounts {
     my $self = shift;
     while (1) {
+        $self->{accounts} = await $self->get_hash_accounts();
```

Comment:
I'm a little surprised this isn't highlighted as the source of the leak in the card comments.